### PR TITLE
[ocean] Fix mappings to index Gerrit data

### PR DESCRIPTION
**Pull request by valeriocos**
This patch prevents to index possible large text fields returned by the Gerrit API, which may cause failures when uploading the data to ES

### CODE REVIEW

**Comment by acs** [comment]
Hmmm, I won't work in possible causes of failures. But let's talk about it! Thanks!

File: grimoire_elk/elk/gerrit.py [hunk 1]
```diff
@@ -45,33 +45,23 @@ def get_elastic_mappings(es_major):
...
-                   "timeopen": {
-                      "type": "double"
-                   }
-                }
+        mapping = """
+        {
+            "properties": {
+               "summary_analyzed": {
+                  "type": "text",
+                  "index": false
```
Comment (acs) [inline grimoire_elk/elk/gerrit.py]: We want to index this field.

**Review by acs — CHANGES_REQUESTED** [review]

File: grimoire_elk/elk/gerrit.py (hunk 1 above)
Comment (valeriocos) [reply grimoire_elk/elk/gerrit.py]: Long summaries may trigger a Lucene exception (Lucene’s term byte-length limit of 32766). Why we have to index this field?

**Review by acs — APPROVED** [review]
LGTM

**Pull request merged**